lib/utils/jsontools.go

```diff
@@ -156,6 +174,11 @@ func WriteYAML(w io.Writer, values interface{}) error {
 	}
 	// first pass makes sure that all values are documents (objects or maps)
 	slice := reflect.ValueOf(values)
+	if slice.Len() == 0 {
+		_, err := w.Write([]byte("[]"))
```

Comment:
there's probably a way to get `reflect` to produce a non-nil slice, but every time I look at those package docs, I don't come away in less than half an hour :) Probably `MakeSlice()` but the way you have it is probably simpler in this specific case.